lib/pghero/methods/connections.rb

```diff
@@ -37,6 +37,58 @@ def connection_sources
             5 DESC, 1, 2, 3, 4
         SQL
       end
+
+      def connection_sources_by_user
+        select_all <<-SQL
+          SELECT
+            datname AS database,
+            usename AS user,
+            COUNT(*) AS total_connections
+          FROM
+            pg_stat_activity
+          GROUP BY
+            1, 2
+          ORDER BY
+            3 DESC, 1, 2
+        SQL
+      end
+
+      def recently_connected_users
+        users = select_all_stats <<-SQL
+          SELECT distinct username
+          FROM "pghero_connection_stats" 
+          WHERE database = #{quote(id)}
+          AND captured_at > date_trunc('day', NOW() - interval '2 hours')
+          ORDER by username
+        SQL
+      end
+
+      def connection_history_for_user(username)
```

Comment:
Would be great to combine this and `recently_connected_users` into a `connection_stats` method that gets stats for all users in a single query.